source/common/tcp_proxy/tcp_proxy.cc

```diff
@@ -138,13 +138,6 @@ Config::Config(const envoy::extensions::filters::network::tcp_proxy::v3::TcpProx
     return drain_manager;
   });
 
-  if (config.has_hidden_envoy_deprecated_deprecated_v1()) {
```

Comment:
I think there may be a bunch of dead code left in tcp_proxy after this, but I can go through and clean it up in a later PR. Very happy to see this going away finally!

Reply:
Sure, please do, this PR is already huge :D